webdriver-spec.html

```diff
@@ -6574,8 +6574,10 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 <p>To get <dfn data-lt="associated cookies">all associated cookies</dfn> to a <a>document</a>,
  the user agent must return the enumerated set of <a>cookies</a>
  that meet the requirements set out in the first step of the algorithm in [[RFC6265]] to
- <a>compute <code>cookie-string</code></a> for an ‘HTTP API’,
- from the <a>cookie store</a> of the given <a>document</a>’s <a>address</a>.
+ <a>compute <code>cookie-string</code></a> for an ‘HTTP API’, from
+ the <a>cookie store</a> of the
+ given <a>document</a>’s <a>address</a>. The returned cookies must
+ include <code>HttpOnly</code> cookies.
```

Comment:
Use `<a>cookie HTTP only</a>` here. We have a reference to it from the relevant RFC.